build

```diff
@@ -101,10 +101,10 @@ Dir.chdir(File.dirname(__FILE__)) do
   add_license(add_deps(output), version)
 
   File.open("pkg/sinon-ie#{version_string}.js", "w") do |f|
-    f.puts(File.read("lib/sinon/util/timers_ie.js"))
+    f.puts(File.read("lib/sinon/util_ie/timers.js"))
```

Comment:
Yea, sure. I figured I'd change as little as possible, so I basically moved `"_ie"` from the file name to the directory name.